components/omnisend/actions/track-event/track-event.mjs

```diff
@@ -0,0 +1,20 @@
+import omnisend from "../../omnisend.app.mjs";
+
+export default {
+  key: "omnisend-track-event",
+  name: "Track Custom Event",
+  description: "Log a custom event for better analytics and targeted marketing. [See the documentation](https://api-docs.omnisend.com/reference/get_events)",
```

Comment:
The name of this component seems confusing to me. I would change the filename to `get-events` and update the name & description.
```suggestion
  key: "omnisend-get-events",
  name: "Get Custom Events",
  description: "Retrieve a list of custom events in Omnisend. [See the documentation](https://api-docs.omnisend.com/reference/get_events)",
```